client/reducers/cohort.js

```diff
@@ -1,96 +1,90 @@
 import { combineReducers } from 'redux';
 
 import {
-    COHORT_REQUEST_LIST,
-    COHORT_REQUEST_LIST_ERROR,
-    COHORT_REQUEST_LIST_SUCCESS,
-    COHORT_CREATE_SUCCESS
+    CREATE_COHORT_SUCCESS,
+    SET_COHORT,
+
+    //
+    //
+    //
+    // TODO
+    //
+    //
+    //
+    // SET_COHORT_ERROR,
+    // SET_COHORT_SUCCESS,
+    // GET_COHORT,
+    // GET_COHORT_ERROR,
+    GET_COHORT_SUCCESS,
+    // GET_USER_COHORTS,
+    GET_USER_COHORTS_SUCCESS
+    // GET_USER_COHORTS_ERROR
 } from '../actions/types';
 
-const defaultState = {
-    index: {},
-    indexOrder: [],
-    indexRequest: {
-        lastRequest: 0,
-        lastResponse: 0,
-        status: 'init'
-    }
+export const initialState = {
```

Comment:
likewise here, I'm not sure we should export this

Reply:
Fixed!